tools/serverpod_cli/lib/src/test_util/builders/method_definition_builder.dart

```diff
@@ -22,6 +23,12 @@ class MethodDefinitionBuilder {
     return this;
   }
 
+  MethodDefinitionBuilder withAnnotations(
+      List<AnnotationDefinition> annotations) {
```

Comment:
Suggestion(NIT): Add comma after parameter.
```suggestion
      List<AnnotationDefinition> annotations,) {
```

When line breaks occur (we use 80 column width) a comma after the last parameter should be used. This helps with formatting.
As you can see we miss it from time to time (looking at the other methods in this file), but we attempt to fix it when we can.